### DISCUSSION TITLE
Expected user behavior on data grid rows that include buttons

### DISCUSSION THREAD

**Discussion by shinytoyrobots** [jpmorganchase/salt-ds#2066] [Feedback]
From discussion with James Sol. Query on what the correct behavior should be - as if a row has a button, then selecting the button also selects the row, and this can be a confusing user experience.

**Comment by mark-tate** [comment]
More of an AG Grid question.
Would imagine we need  `event.stopPropagation()` if we don't want the row selected.
Closing due to in-activity, best to raise as an internal Stackoverflow if still an issue.
